include/symbol.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <cstddef>
+
+namespace OS {
+
+	template <typename T>
+	class Symbol {
+	public:
+		Symbol(std::shared_ptr<Process> process, size_t offset) :
```

Comment:
We can go cool mode and add const to relevant method parameters. Change everywhere relevant